### PR TITLE
Added Auto scrolling background component

### LINKED ISSUE #164: Automatic Scrolling Background
### Feature request-
This could be used in various ways to showcase stuff to the users. Examples: testimonials, key features, etc.

https://user-images.githubusercontent.com/73700530/125194844-ab1dbf00-e270-11eb-950e-85b2db188ab5.mov

LGM-SOC

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

### PR DESCRIPTION
**Pull request by Anupam-Panwar**
## What is the change?
Added a directory named Auto-Scrolling-Background in HTML+CSS+JS/Background

## Related issue?
closes: #164 

## How was it tested?
Component was thoroughly tested in Chrome Browser

## Checklist:
-   [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
-   [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
-   [x] Have you made corresponding changes to the documentation?
-   [x] Have you tested the code before submission?
-   [x] Did you lint and format your code before making the Pull Request?
-   [x] Have you made the GIF for the component you added?

## GIF you created:
auto scroll background

### CODE REVIEW

**Comment by Pranav016** [comment]
> ## What is the change?
> Added a directory named Auto-Scrolling-Background in HTML+CSS+JS/Background
> 
> ## Related issue?
> closes: #164
> 
> ## How was it tested?
> Component was thoroughly tested in Chrome Browser
> 
> ## Checklist:
> * [x]  Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
> * [x]  Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
> * [x]  Have you made corresponding changes to the documentation?
> * [x]  Have you tested the code before submission?
> * [x]  Did you lint and format your code before making the Pull Request?
> * [x]  Have you made the GIF for the component you added?
> 
> ## GIF you created:
> auto scroll background

I think people might not be able to tell any scrolling in this gif. Maybe change some background colors of sections

**Comment by Anupam-Panwar** [comment]
@Pranav016 
I have incorporated the suggested changes

File: HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
...MtrG80CLyQK2JEyQ6nVRkWOKLCv9Ua1vNED3DdIJyt5sPt4D443MhDCVyXGTGg25+ohKye64+O8+FhuH9MUoZm9Sh6IPuY3QTpBv5sTC40OOw3l8S5mEt6ACguFdDlRsFlQ86/Hc4fxAVxV+bvgq01xmhvupNx8PFsuwQwy2vnS7C65XkADQGTew0CTdxoVF32nrY6qfUnoSN/UOn+aj6m36TX0fJVmfnvudVHACaH5ReogVUUxBxiEsQyL5awSazSpeiahTZY6NY5TJPebvjyn1fvciEgUOQ+5rEvc1uiQz6JkDkUuHZtkHtv7oQvrUr9ziptzhyxpkfIh/4u4JYwo2strPLySt6wXkryfGOVtv0GebUB7Yn38CdjlE6NiYOvrSRWFhhxrA7v9jKt8oZy8xeL3XEMSaRC5uZrOuKdbqPaplfqz0UntF/Ih31vVV5VMGGTYXQkI4GveZs4nOxiL6xf+llaP8+xsl4u035bVQYUyJw6Xsck3IFxb+Jet0uzYX8nCCqp0Yx142ry/68qsIDGnDo+m8S2aOkzHgWNmw4aYKGxJhEH/E9U8+KegM1z0CB5OyRIQrOGltck3Xt0At22KfN/q7pXQpkDRus005Jd5CCZMNd+vxYWGrzha3tqYSUvsct6IlofJ/7SDJZeOKm1lVpYb/mKI60eFq0uMgIYRaeP6f3jbH3WFh7qRkPC6g37oNYAS4MkdGcMaDqaqiusCI3f066aYJGRbNcrLZxc1JbzK7AQHr3fMJip+aWu4UAo9zSsXy7DQvfL+71kjKnldcHmkU+rd4KF0P5d31FK1fYiapcYV06iYbE3Dwuj++SN36zcCovmx2cLVo9/CitCp+vbTZ1FPdkL63oPJ3vP2K9WWBg5x3d+YXeip38dxTU3U1qTQIv60CGYIlrNmD7Md/87A89h0Vy0ORmeDbvh9SFgY/fOh8nybeMFTl1y/mQe7V3iphhD8y8e+Bl/26ljgaRpHp396ieQ0jrDUqJSsCSkYElIGtZPZqssS0IKloQULAkpWBJSsCSkYElIwZKQgiUhBUtCCpaEFCwJKVgSUrAkpGBJSMGSkIIlIQVLQgqWhBQsCSlYElKwJKRgSUjBkpCCJSEFS0IKloQULAkpWBJSsCSkYElIwZKQgiUhBUtCoPy3gZRa9B+ovCyGraEZmgAAAABJRU5ErkJggg==")
```
Comment (theBeginner86) [inline HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css]: @Pranav016 What do you think regarding this? Shall we go for a local image from component folder itself or this seems fine?

**Review by theBeginner86 — COMMENTED** [review]
@Anupam-Panwar 
Everything looks good 👍 
But I was thinking that there isn't any need of using such a long URL for setting a background image. You can very easily set any local image from the assets folder (after adding the `asset folder` in the `HTML+CSS+JS/Background/Auto-Scrolling-Background` and then using images from there. 
This might also make it difficult from a user perspective, in the sense that they might not be able to reuse it that easily as they would have been by replacing the URL from the local folder.

Even then, let us wait for the views from @Pranav016.

**Comment by Pranav016** [comment]
> @Anupam-Panwar
> Everything looks good 👍
> But I was thinking that there isn't any need of using such a long URL for setting a background image. You can very easily set any local image from the assets folder (after adding the `asset folder` in the `HTML+CSS+JS/Background/Auto-Scrolling-Background` and then using images from there.
> This might also make it difficult from a user perspective, in the sense that they might not be able to reuse it that easily as they would have been by replacing the URL from the local folder.
> 
> Even then, let us wait for the views from @Pranav016.

I agree with this review
@Anupam-Panwar can you use another image with a shorter link? That would do the job 👍

File: HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css (hunk 1 above)
Comment (Anupam-Panwar) [reply HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css]: Will incorporate this change asap
I am thinking of using this image now
image

Should I go with this ?

Comment (theBeginner86) [reply HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css]: ok. Try with it.

File: HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+.container {
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+  position: relative;
+}
+
+.sliding-background {
+    z-index: -1;
+  background: url("../../../assets/MUI\ Logo.png")
```
Comment (theBeginner86) [inline HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css]: Thanks, @Anupam-Panwar :)
Actually, I meant that you first add the assets folder in `HTML+CSS+JS/Background/Auto-Scrolling-Background` then add the image in `HTML+CSS+JS/Background/Auto-Scrolling-Background/assets` folder. Then use the image from there. Like this everything related to the component will remain in the same directory.
Can you please make this final change?

**Review by theBeginner86 — CHANGES_REQUESTED** [review]

File: HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css (hunk 2 above)
Comment (Anupam-Panwar) [reply HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css]: Okay @theBeginner86 Will do it

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**